libs/SmartStore/SmartStoreTests/SFSmartSqlWithExternalStorageTests.m

```diff
@@ -30,6 +30,29 @@
 #import <SalesforceSDKCore/SFJsonUtils.h>
 #import "SFSoupSpec.h"
 
+@interface SFSmartSqlTests ()
```

Comment:
That makes sense, I just made the change here: https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1650/commits/d8b65aacd8f24d6e52c4b058d24829de22283783